tdvt/tdvt/tdvt_core.py

```diff
@@ -563,48 +588,94 @@ def get_csv_row_data(tds_name, test_name, test_path, test_result: TestResult, te
         error_msg = test_result.saved_error_message if test_result.saved_error_message else error_msg
         error_type = test_result.get_error_type()
 
-    columns = [suite, test_set_name, tds_name, test_name, test_path, str(passed), str(matched_expected),
-               str(diff_count), test_case_name, test_type, priority, categories, functions, cmd_output,
-               str(error_msg), str(error_type), float(case.execution_time), generated_sql, actual_tuples,
-               expected_tuples]
+    if is_perf_run:
```

Comment:
Could these duplicated columns parts be more closely encapsulated with the code in constants.py? tdvt_output.py? I'm not sure if this is practical or helps though.

Reply:
I'll open a follow-up for that. We should do it; it'll require some plumbing work that I will want to test pretty thoroughly.